components/layout/sidebar.js

```diff
@@ -71,18 +86,36 @@ export default function SideBar() {
           className="border-t-2 border-white bg-blue"
           popperArrow
         >
-          <MenuItem
-            icon={<FontAwesomeIcon icon={faGem} className="cursor-pointer" />}
-          >
-            Dashboard
+          <MenuItem>
+            <Link href="/">
+              <a>DASHBOARD</a>
+            </Link>
           </MenuItem>
-          <SubMenu
-            title="Components"
-            icon={<FontAwesomeIcon icon={faHeart} className="cursor-pointer" />}
-          >
-            <MenuItem>Component 1</MenuItem>
-            <MenuItem>Component 2</MenuItem>
-          </SubMenu>
+          {subjects &&
+            subjects.map((s) => (
```

Comment:
we should avoid single name variables.  I know you used them in a previous PR, but it was all on one line so I didn't think it was a big deal.  But there is no cost to writing out the variable names and is much more readable.
```
subjects.map((subject) => (
  title={`${subject.name} - ${subject.grade}`}
```

same for 
```
subject.units.map((unit) => (
```

Reply:
also, generally, it's rarely a good practice to use single variable names when working on a team, unless there are exceptions made or the team has decided in particular cases that single variable names don't matter in certain contexts (like the one I mentioned above)

Reply:
ok, will do